NitroxServer/Serialization/World/WorldData.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NitroxModel.DataStructures;
+using NitroxServer.GameLogic;
+using NitroxServer.GameLogic.Bases;
+using NitroxServer.GameLogic.Entities;
+using NitroxServer.GameLogic.Items;
+using NitroxServer.GameLogic.Vehicles;
+using ProtoBufNet;
+
+namespace NitroxServer.Serialization.World
+{
+    [ProtoContract]
+    public class WorldData
+    {
+        [ProtoMember(1)]
+        public List<Int3> ParsedBatchCells { get; set; }
+
+        [ProtoMember(2)]
+        public DateTime ServerStartTime { get; set; }
```

Comment:
DateTime is a struct so it won't be null.
```suggestion
        public DateTime? ServerStartTime { get; set; }
```

Reply:
I don't think you ever want this to be `null`? Though it shouldn't be a wrong or invalid date either.

Reply:
Yup I should have said a bit more on why I wanted the change. But right now, because it can't be `null`, it'll potentially be a bad `DateTime` if serialization has no value for it. Either that or it throws an exception.

Reply:
@Measurity I believe the date is January 1st some year... I cant remember the year though

Reply:
it will default to 01/01/0001

Reply:
It'd still prefer it would be `DateTime?` because 01/01/0001 is a valid date but we want the value to say "I'm not a valid date" hence, `null`. See: https://stackoverflow.com/questions/4821560/better-datetime-or-use-defaultdatetime-for-null

Reply:
unfortunately it will never be null through protobuf... i am rather bad at memory but i believe it handles DateTime differently

Reply:
That seems a good concession to make, given that it will either be assigned to something useful or remain `null`. Using the nullable type forces this to be explicitly checked on retrieval as well.

Reply:
@killzoms the nullable type forces it to be wrapped in something that can represent null. It seems obvious that a datetime is converted to an integer or timestamp-string of some sorts.

Reply:
Yeah it's a struct so it can't be null. I thought that protobuf handled Nullable<T> too but if that's not the case then `default(DateTime)` it is.